deploy.js

```diff
@@ -0,0 +1,56 @@
+const zipFolder = require('zip-folder');
+const editJsonFile = require("edit-json-file");
+const fs = require('fs');
+
+let file = editJsonFile('./dist/manifest.json');
```

Comment:
Why `let` and not `const`?  Here and elsewhere.

Reply:
Also below.  I didn't see any `let` that needed to be there so the var could be mutated later...